weather_mv/loader_pipeline/bq.py

```diff
@@ -243,6 +261,7 @@ def expand(self, paths):
         """Extract rows of variables from data paths into a BigQuery table."""
         extracted_rows = (
                 paths
+                | 'CreateTable' >> beam.Map(self.create_bq_table)
```

Comment:
Concern: I am pretty sure this will try to create a BQ table for every element of `paths`. Remember, `self.tables` won't refer to the state of the class, since global state is not really a think for parallel steps like this.

Reply:
Consider using something like `sample`: https://beam.apache.org/documentation/transforms/python/aggregation/sample/